File: Lab7/src/modules/urls.js
class Urls {
    constructor() {
        this.url = 'http://localhost:8000/api/';
    }

    langs() {
        return `${this.url}lang/`
    }

    lang(id) {
        return `${this.url}lang/${id}/`
    }
}

export const urls = new Urls()

File: Lab6/Lab6/main_app/urls.py
from django.urls import path, include
from . import views
from rest_framework import routers


router = routers.DefaultRouter()
router.register(r'lang', views.LangViewSet)


urlpatterns = [
    path('', views.index),
    path('api/', include(router.urls)),
    path('<str:model_name>/', views.list, name='list')
]